### ISSUE TITLE
/tmp directory created on plugin launch

### ISSUE THREAD

**Issue by hellocatfood** [greatest-ape/OctaSine#32]
**Describe the bug**
When the plugin is launched a /tmp folder is created in my home folder (`/home/user`). Inside the folder is a file called `OctaSine.log` which contains only the following:

```
23:36:21 [INFO] init
23:36:21 [INFO] OS: Ubuntu 21.10 (impish) [64-bit]
23:36:21 [INFO] OctaSine build: v0.5.3 (35957f4) (wgpu)
23:36:21 [INFO] Couldn't load settings: No such file or directory (os error 2)
```

**What did you try to do**
1. Open Ardour or Carla
2. Add the Octasine plugin

**Expected behavior**
I didn't expect to see a folder created, especially not in the home directory

**Software (please complete the following information):**
 - OctaSine version 0.5.3
 - DAW Ardour and Carla
 - OS: Ubuntu 21.10

**Additional context**

**Log file contents***
Please include the contents of the OctaSine log file. It is a file called `OctaSine.log` in a folder called `tmp` inside of your home folder. Your home folder is the one which typically contains folders such as `Downloads` and `Documents`.

```
Please replace this text with the log file contents.
```

**Comment by hellocatfood** [comment]
So in filling out this bug report I see that the tmp folder being created is by design! Closing this now

**Comment by hellocatfood** [comment]
Is there any chance that this folder could be moved to a hidden folder i.e. something like `.octasine`?

**Comment by greatest-ape** [comment]
Yes! In the upcoming release, a different folder will be used (~/.config/octasine on Linux.)

**Comment by hellocatfood** [comment]
Great, thank you!
